site/guide/user-management.qmd

```diff
@@ -0,0 +1,66 @@
+---
+title: "User management"
+keywords: "user management, RBAC, roles, groups, permissions"
+---
+
+## Introduction 
+
+User management involves controlling and organizing user accounts within the ValidMind Platform UI. It encompasses tasks like user creation, authentication, authorization, and access control. Effective user management is essential for maintaining security and operational efficiency. 
```

Comment:
Note for myself: we probably want to update this text to reflect the tasks we actually cover more closely, e.g. mention groups, roles, and permissions along with users, and tasks like inviting new users.